tsfile/src/main/java/org/apache/iotdb/tsfile/read/TsFileSequenceReader.java

```diff
@@ -1142,6 +1145,9 @@ public long selfCheck(
           this.position(),
           e.getMessage());
     }
+    if (loadLastChunkMetadata) {
+      chunkGroupMetadataList.add(new ChunkGroupMetadata(lastDeviceId, chunkMetadataList));
+    }
```

Comment:
What if the last ChunkGroupMetadata has already been added before.

Reply:
resolve